client/src/component/admin/MenuForm.js

```diff
@@ -76,8 +76,9 @@ function MenuForm({ handleItem, values, handleClose, buttonText = "Add" }) {
                   value={inputValues.price}
                   onChange={handleChange}
                 />
-                <FormLabel htmlFor="price">Select Category</FormLabel>
+                <FormLabel htmlFor="categoryId">Select Category</FormLabel>
                 <Select
+                  id="categoryId"
```

Comment:
This is the bug I mentioned. The label wasn't associated with the correct field.